src/test/java/com/wikia/webdriver/elements/mercury/pages/InfoboxBuilderPage.java

```diff
@@ -136,9 +304,36 @@ public int countHeaders() {
     return headers.size();
   }
 
-  public String getBackgroundColor() {
-    wait.forElementVisible(component.get(0));
-    return component.get(0).getCssValue("background-color");
+  public InfoboxBuilderPage changeHeaderCollapsibilityState(int index) {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
```

Comment:
Can you explain why? It's pseudoelement on :after and my only idea how to test it was by executing js here. Or if this remark is only about casting driver to JS, please suggest how I should implement that one.

Thanks!